lib/dry/cli/command_registry.rb

```diff
@@ -206,6 +205,33 @@ def after_callbacks
           @node.after_callbacks
         end
       end
+
+      # Stack for callbacks
+      #
+      # @since 0.4.0
+      # @api private
+      class CallbackStack
```

Comment:
I wouldn’t call it Stack because it isn’t, what do you think if we call it Chain like in Hanami utils?